app/views/how-tos/layouts.html

```diff
@@ -0,0 +1,100 @@
+<p><code>{{'{% extends "layouts/admin.html" %}' | escape }}</code></p>
+<p>The Prototype Kit comes with a layout file for you to edit. You can also add more layouts if you need to.</p>
+<h2 id="adding-layouts">Adding layouts</h2>
+<p>In your code editor, open <code>app/views/layouts.html</code>.</p>
+<p>Note the line:</p>
+<pre tabindex="0" class="app-pre"><code class="language-markup">{{'{% extends "template.html" %}' | escape }}
+</code></pre>
+<p>It means this layout extends a standard layout that comes with the Prototype Kit. It loads the default code needed for NHS.UK branded pages, along with the functionality in the kit, such as automatically storing data.</p>
+<p>You can make changes to existing blocks and define your own blocks in your layout.</p>
+<h2 id="unbranded-pages">Unbranded pages</h2>
+<p>If you do not want to use the NHS.UK logo or footer, you can choose to use unbranded pages in your prototype. </p>
+<h2 id="using-blocks">Using blocks</h2>
+<p>Blocks are how layouts and pages share code. For example, there is a block called <code>header</code> for the header content on every page.</p>
+<p>These are some of the default blocks on the <a href="https://service-manual.nhs.uk/design-system/styles/page-template">template page on the NHS.UK Design System</a>.</p>
+<h3 id="header-block">Header block</h3>
+<p>You can make changes to the existing NHS.UK header using the <code>header</code> block. This example adds navigation:</p>
+<pre tabindex="0" class="app-pre"><code class="language-markup">{{'{% block header %}
+{{ header({
+  transactionalService: {
+      name: "Find your NHS number",
+      href: "https://www.nhs.uk/nhs-services/online-services/find-nhs-number/"
+    },
+    showNav: "true",
+    showSearch: "false",
+    primaryLinks: [
+    {
+        href: "#1",
+        text: "Navigation item 1",
+        active: true
+      },
+      {
+        href: "#2",
+        text: "Navigation item 2"
+      },
+      {
+        href: "#3",
+        text: "Navigation item 3"
+      }
+    ]
+  })
+}}
+{% endblock %}' | escape }}
+</code></pre>
+<p>Read more about <a href="https://service-manual.nhs.uk/design-system/components/header">headers in the NHS.UK Design System</a>.</p>
+<h3 id="footer-block">Footer block</h3>
+<p>You can make changes to the existing NHS footer using the <code>footer</code> block:</p>
+<pre class="app-pre"><code class="language-markup">{{'{% block footer %} 
+{{ footer({
+meta: {
+ items: [
+   {
+     href: "/privacy",
+     text: "Privacy policy"
+   },
+   {
+     href: "/manage-prototype",
+     text: "Manage your prototype"
+   },
+   {
+     href: "/manage-prototype/clear-data",
+     text: "Clear data"
+   }
+ ],
+ visuallyHiddenTitle: "Footer links"
+}
+}) }}
+{% endblock %}' | escape }}
+</code></pre>
+<p>Read more about <a href="https://service-manual.nhs.uk/design-system/components/footer">footers in the NHS.UK Design System</a>.</p>
```

Comment:
I don't think we need to include these examples of headers and footers - better to point to the pages in the service manual for that (especially as they're about to change).

Maybe reduce this down to just introducing the concept of blocks?

```suggestion
<h2 id="using-blocks">Using blocks</h2>

<p>Blocks are named sections of a page.</p>

<p>For example, in the <a href="https://service-manual.nhs.uk/design-system/styles/page-template">NHS page template</p> there are blocks defined for <code>header</code>, <code>content</code> and <code>footer</code>.

<p>You can use these in your layouts to set different areas of the page.</p>

<p>See the <a href="https://service-manual.nhs.uk/design-system/styles/page-template#options">NHS page template options</a> for a full list of all the defined blocks.</p>
```